plugins/org.locationtech.udig.tutorials.tool-view/src/org/locationtech/udig/tutorials/toolview/View.java

```diff
@@ -83,6 +84,11 @@ public IStatusLineManager getStatusLineManager() {
         return getViewSite().getActionBars().getStatusLineManager();
     }
 
+    @Override
+    public MapSite getMapSite() {
+        return null;
```

Comment:
like in other tutorials, a mapSite instance should be created in createPartControl and used right here to access it:

mapSite = new MapSite(getViewSite(), this);

Reply:
see above